docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+
+An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+IMPORTANT: Before enabling a threat intelligence feed, ensure you have installed a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.
+
+. Choose one of the following:
+.. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.
+.. From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
+. Select a threat intelligence integration, then complete the installation steps.
++
+NOTE: Refer to the Threat intelligence section of the https://docs.elastic.co/integrations[Elastic integration documentation] for more information about required field mappings.
+
+. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[intelligence-page-ui]]
+== Intelligence page UI
+
+After you add indicators to the Intelligence page, you can <<examine-indicator-details,examine indicators>> and search, filter, and take actions on indicator data. Indicators are also visualized in the Trend view, which shows the occurrence of indicators over time.
+
+[role="screenshot"]
+image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]
+
+[discrete]
+[[examine-indicator-details]]
+=== Examine indicator details
+Learn more about an indicator by clicking the *View details* button and opening the Indicator details flyout. The flyout contains these informational tabs:
+
+* *Overview*: Offers a summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
++
+NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The *TLP Marking* field will be empty if the data is not available.
+
+* *Table*: The indicator data in table format. Data is organized into field-value pairs.
```

Comment:
Nit: It feels unnecessary to mention the field-value pairs detail; it seems pretty obvious, especially if the table has headings saying "Field" and "Value" (the screenshot shows the Overview tab but I assume Table tab is similar?)

Reply:
Good point and yeah the **Table** tab is nearly identical.

Reply:
```suggestion
* *Table*: The indicator data in table format. 
```